README.rst

```diff
@@ -80,73 +99,86 @@ If we have the following secondary whitelist list:
     REG face
     RZD example
 
-our system will actually :
+UHBW will do as follows:
 
 * Remove every line which match :code:`facebook.com` and :code:`www.facebook.com`
-* Remove everyline which match :code:`example.*`
-* In complementary convert all lines with :code:`ALL ` or :code:`REG` to the right format.
-* Check every line against the regular expression.
-* Print or save on screen the results.
+* In complementary convert all lines with :code:`ALL ` or :code:`REG` to the 
+  right format.
+* Remove every line which match :code:`example.*`
+* Check every line against the regular expression. More about this in next chapter.
+* Print the results on screen or save to output file :code:`-o $output.file`.
 
-The generated regular expression will be in this example:
+The generated regular expression will from this example be:
 
 ::
 
     (\.gov$)|(face)|(example(.*))
 
 
 .. note::
-    The :code:`example` group is much longer as we construct the list of TDL based on the Root Zone Database of the IANA and the Public Suffix List project.**
+    The :code:`example` group is much longer, as we construct the list of TDL 
+    based on the Root Zone Database, of IANA and the Public Suffix List 
+    project.**
 
-Which actually means that we whitelist:
+Which means UHBW actually will whitelist:
 
 * all elements/lines which ends with :code:`.gov`
 * all elements/lines which contain the word :code:`face`
-* all possible TDL combinaison which starts with :code:`example`
-
-Contributors
-------------
-
-* Daniel - `@dnmTX`_
+* all possible TDL combination which starts with :code:`example`
 
 Usage of the tool
 -----------------
 
-The sript can be called as :code:`uhb-whitelist`, :code:`uhb_whitelist` and :code:`ultimate-hosts-blacklist-whitelist`.
+The script can be called by :code:`uhb-whitelist`, :code:`uhb_whitelist` or 
+:code:`ultimate-hosts-blacklist-whitelist`.
 
 ::
 
     usage: uhb_whitelist [-h] [-a ANTI_WHITELIST [ANTI_WHITELIST ...]] [-d]
-                        [-f FILE] [-o OUTPUT] [-m] [-p PROCESSES] [-v]
-                        [-w WHITELIST [WHITELIST ...]] [-wc]
+			 [-f FILE] [-o OUTPUT] [-m] [-p PROCESSES] [-v]
+                         [-w WHITELIST [WHITELIST ...]] [-wc]
 
-    The tool to clean a list or a hosts file with the Ultimate Hosts Blacklist
-    whitelist list or your own.
+    UHBW is a tool to clean up lists or hosts files with the hosted and/or your 
```

Comment:
@spirillen Please change in the source code this part. ==> https://github.com/Ultimate-Hosts-Blacklist/whitelist/blob/4b7343eb25f359a1f2693763238ae89157751681/ultimate_hosts_blacklist/whitelist/__init__.py#L169-L332

Reply:
Clarification: It's generated by argparse (`--help`), it's just a script which updates this section. So changes should be made in the code itself.

Reply:
Sure, but that isn't today ping me later if I forget it (I'm not that fresh today. Maybe I've someone from China :rofl: )

Reply:
Submitted, what command would I use to generate a (pre-)build of this so I could verify the output locally? I most concerned about the layout in the `--help` output in a terminal